site/docs/tutorials/using-various-svg-loader.md

```diff
@@ -121,7 +121,7 @@ module.exports = {
 								test: /\.svg(\?v=\d+\.\d+\.\d+)?$/,
 								use: [
 									{
-										loader: fileLoader,
+										loader: 'file-loader',
```

Comment:
Why is this necessary? By using the `fileLoader` export, we actually ensure that we are using the file loader installed by wpackio/script. Otherwise this can cause issues when using with other tooling.